presenter/src/components/inputs/ContactForm.vue

```diff
@@ -127,30 +127,24 @@ const dataprivacy = ref(0)
 
 const form = ref<HTMLFormElement>()
 
-const { mutate: sendContactForm, onDone, onError } = useMutation(createContactFormMutation)
-
-onDone(() => {
-  // eslint-disable-next-line no-console
-  console.log('successfully sent form')
-})
-
-// eslint-disable-next-line promise/prefer-await-to-callbacks
-onError((err) => {
-  // eslint-disable-next-line no-console
-  console.log(err.message)
-})
+const { mutate: sendContactForm } = useMutation(createContactFormMutation)
 
 async function submitForm() {
   const isValid = await form.value?.validate()
   if (isValid?.valid) {
-    await sendContactForm({
+    sendContactForm({
       data: {
         firstName: firstname.value,
         lastName: lastname.value,
         email: email.value,
         content: message.value,
       },
     })
+      .then(() => {
+        form.value?.reset()
+        return undefined
+      })
+      .catch(() => {})
```

Comment:
```suggestion
    try {
      await sendContactForm({
        data: {
          firstName: firstname.value,
          lastName: lastname.value,
          email: email.value,
          content: message.value,
        },
      })
      form.value?.reset()
    } catch(error){}
```

I think await syntax would be more consistent - not a blocker tho